redisinsight/ui/src/assets/img/icons/live-time-recommendations-active.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="18" height="24" viewBox="0 0 18 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<mask id="path-1-inside-1_145_9122" fill="white">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M13.6157 15.7502C13.6157 15.4165 13.7848 15.1082 14.0496 14.9052C16.0305 13.3868 17.3078 10.9964 17.3078 8.30769C17.3078 3.71948 13.5883 0 9.00007 0C4.41186 0 0.692383 3.71948 0.692383 8.30769C0.692383 10.9966 1.96985 13.3872 3.95099 14.9055C4.21585 15.1085 4.38496 15.4169 4.38496 15.7506V19.3076C4.38496 19.8599 4.83268 20.3076 5.38496 20.3076H12.6157C13.168 20.3076 13.6157 19.8599 13.6157 19.3076V15.7502Z"/>
+</mask>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M13.6157 15.7502C13.6157 15.4165 13.7848 15.1082 14.0496 14.9052C16.0305 13.3868 17.3078 10.9964 17.3078 8.30769C17.3078 3.71948 13.5883 0 9.00007 0C4.41186 0 0.692383 3.71948 0.692383 8.30769C0.692383 10.9966 1.96985 13.3872 3.95099 14.9055C4.21585 15.1085 4.38496 15.4169 4.38496 15.7506V19.3076C4.38496 19.8599 4.83268 20.3076 5.38496 20.3076H12.6157C13.168 20.3076 13.6157 19.8599 13.6157 19.3076V15.7502Z" fill="#FFAF2B"/>
```

Comment:
you should remove fill from svg's which you want to use as ReactComponent to override colors.
Then you don't need to override them over styles